tests/assets/eks_service_role_trust_policy.json

```diff
@@ -0,0 +1,16 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "Service": [
+                    "eks-gamma.aws.internal",
+                    "eks.amazonaws.com",
+                    "eks-beta.aws.internal"
```

Comment:
@hakuna-matatah Any concerns? I'd defer to you.

Reply:
this file is not getting used so removed it.